astro/src/content/docs/lifecycle/register-users/advanced-registration-forms.mdx

```diff
@@ -106,7 +107,7 @@ The following fields are available by default:
 * Mobile phone
 * Birthdate
 
-If you need additional fields, you must create them. To do so, navigate to <strong>Customizations -> Form Fields</strong>. You'll see a list of the above default fields, any existing custom fields and a button to create new ones. 
+If you need additional fields, you must create them. To do so, navigate to <Breadcrumb>Customizations -> Form Fields</Breadcrumb>. You'll see a list of the above default fields, any existing custom fields and a button to create new ones.
```

Comment:
We should use Oxford commas.

```suggestion
If you need additional fields, you must create them. To do so, navigate to <Breadcrumb>Customizations -> Form Fields</Breadcrumb>. You'll see a list of the above default fields, any existing custom fields, and a button to create new ones.
```

Reply:
Linter induced change (not me), but fixed anyways.